docs/proposals/podpolicy.md

```diff
@@ -0,0 +1,149 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.1/docs/proposals/choosing-scheduler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# PodPolicy and its use in choosing the scheduler in a multi-scheduler system
+
+## Introduction
+
+This document describes a new API resource, `PodPolicy`, for defining policies that apply to pods
+based on the pods' characteristics. Its initial use is to serve as the configuration for a
+new admission controller that will assign the scheduler name annotation in a
+[multi-scheduler](multiple-schedulers.md) Kubernetes system.
+
+## API
+
+```go
+// PodPolicySpec defines the configuration of the PodPolicy API resource.
+// Each rule is tested in order, and the first one that matches is applied.
+// To specify a default policy, make the last rule have a PolicyPredicate containing
+// an empty PodSelector (matches all pods in the namespace).
+type PodPolicySpec struct {
+	// A list of rules that define the policy. Each rule is tested in order, and the first one
+	// that matches is applied. 
+	Rules []PodPolicyRule `json:"rules,omitempty"`
+}
+
+// If the PolicyPredicate is met, then all of the actions specified
+// by the remaining fields are applied. 
+type PodPolicyRule struct {
+	PolicyPredicate *PolicyPredicate `json:"policyPredicate,omitempty"`
+	// Add the specified annotations to the pod.
+	Annotations map[string]string  `json:annotations,omitempty`
```

Comment:
Here `Annotations` is too specific.
IIUC, there should be a list of actions.
Maybe define something like:
```Go
Actions []ActionOnPod
...
type ActionOnPod struct {
  Annotations ...
}
```